assets/parameters.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I only included this in here so that you all could see what I was specifying when running to workflow. Don't intend for it to stay there. Probably would have been better in the README!